projects/components/src/paginator/paginator.component.ts

```diff
@@ -64,12 +64,24 @@ export class PaginatorComponent implements OnChanges, PaginationProvider {
   public pageSizeOptions: number[] = [25, 50, 100];
 
   @Input()
-  public pageIndex: number = 0;
+  public set pageIndex(pageIndex: number) {
+    this._pageIndex = pageIndex;
+  }
+  public get pageIndex(): number {
+    return this._pageIndex ?? 0;
+  }
+  private _pageIndex?: number;
 
   @Input()
-  public pageSize: number = 50;
+  public set pageSize(pageSize: number) {
```

Comment:
If I pass a random page size, would it appear on the available page size options dropdown?

Reply:
It probably would not - although that's an existing issue in component, it's a little more obvious now. I can probably merge in the actual pageSize with the options if we want to - or reject a page size that's not in the options.

Reply:
Yeah. I was thinking we should merge it in the available options. Rejecting it would make it a little rigid.

Reply:
OK, will add that in - mind if it's a separate PR?

Reply:
No Sir!